packages/bot-web-ui/src/components/dashboard/dashboard.tsx

```diff
@@ -66,16 +65,18 @@ const Dashboard = ({ active_tab, setActiveTab, toggleStrategyModal, is_drawer_op
                     </Tabs>
                 </div>
             </div>
-            <div className='dashboard__run-strategy-wrapper'>
-                <RunStrategy />
-                {/*
+            {!isMobile() && (
+                <div className='dashboard__run-strategy-wrapper'>
+                    <RunStrategy />
+                    {/*
                     1. Bot-Builder
                     2. Quick Strategy
                     3. Charts
                 */}
```

Comment:
```suggestion
                    */}
```

Also add one indent for the list items inside the comment for prettier comments.